lib/whitehall_importer/create_edition.rb

```diff
@@ -1,17 +1,19 @@
 module WhitehallImporter
   class CreateEdition
-    attr_reader :document_import, :current, :whitehall_edition, :edition_number, :user_ids
+    attr_reader :document_import, :whitehall_edition, :change_history, :current, :edition_number, :user_ids
 
     def self.call(*args)
       new(*args).call
     end
 
-    def initialize(document_import:, whitehall_edition:, current: true, edition_number: 1, user_ids: {})
+    def initialize(document_import:, whitehall_edition:, change_history:,
+                   current: true, edition_number: 1, user_ids: {})
       @document_import = document_import
       @current = current
       @whitehall_edition = whitehall_edition
       @edition_number = edition_number
       @user_ids = user_ids
+      @change_history = change_history
```

Comment:
Minor, and not your fault, but would be nice if these matched param ordering.

Reply:
Yep sure, I've also made sure the ordering matches up in the import file which passes these arguments over!